app/routes/index.js

```diff
@@ -117,34 +123,46 @@ router.get("/users/:id/edit", isLoggedIn, (req, res) => {
     if(req.params.id === undefined) return res.send(500)
     if(!validator.isAlphanumeric(req.params.id)) return res.sendStatus(500)
 
-     User.findById(req.params.id, (err, foundUser) => { 
+    User.findById(req.params.id, (err, foundUser) => { 
         if(err){
             req.flash("error", "Oops! Something went wrong!")
             console.log(err)
             return res.redirect('/')
         } else {
         res.render("users/edit", {user: foundUser})
         }
-     })
+    })
 })
 
 //Update ROUTE
 router.put("/users/:id", isLoggedIn, (req, res) => {
-    if(req.params.id === undefined) return res.send(500)
-    if(!validator.isAlphanumeric(req.params.id)) return res.sendStatus(500)
+    const email = req.body?.email ?? null
+    let avatar = req.files?.avatar ?? null
+    const bio = req.body?.bio ?? null
+    let avatarPath = null
```

Comment:
Why is it declared here instead of the block where it's being created.

Reply:
Leftover effect of code that was changed, I'll fix